multibranch/src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java

```diff
@@ -144,171 +104,114 @@ private static void assertRevisionAction(WorkflowRun build) {
     }
 
     @Test public void exactRevisionSubversion() throws Exception {
-        story.addStep(new Statement() {
```

Comment:
There are two commits in the PR. It will probably make more sense to read them individually.